test/asynciotest.c

```diff
@@ -339,7 +339,7 @@ static int test_asyncio(int test)
          * Write some test data. It should never take more than 2 attempts
          * (the first one might be a retryable fail).
          */
-        for (ret = -1, i = 0, len = 0; len != sizeof(testdata) && i < 2;
+        for (i = 0, len = 0; len != sizeof(testdata) && i < 2;
```

Comment:
With this change made, we can join the following line onto this one to make the `for` a single line